lib/services/auth/auth-service.js

```diff
@@ -0,0 +1,135 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+var jwt = require('jsonwebtoken');
+var crypto = require('crypto');
+
+module.exports = authServiceFactory;
+di.annotate(authServiceFactory, new di.Provide('Auth.Services'));
+di.annotate(authServiceFactory,
+    new di.Inject(
+        'Services.Configuration'
+    )
+);
+function authServiceFactory(
+    configuration
+) {
+    function AuthService() {
+    }
+
+    AuthService.prototype.init = function(){
+        this.algorithm = 'HS256';
+        this.secretOrKey = 'RackHDRocks!';
```

Comment:
This is actually hard coded here. Are you suggesting that user may want to change it to some other secret and we got need to offer them a easier way to do that? In that case, I would suggest to add a item in monorail.json called tokenSecret.